index.rst

```diff
@@ -4,54 +4,146 @@
+encountered to date), the instrumental PSF may begin to have an effect. To date, the PSF quality is dominated by dome
+seeing and tracking error.
+
+In correcting the collimation/focus using the hexapod it is important to understand the limits of motion. Motion of the
+hexapod and how it relates to WFE correction is defined by a sensitivity matrix. The derivation of this matrix is
+discussed in detail in `this technote <tstn-016.lsst.io>`_. For the purposes of this document, a user must know how the
+physical limitations of the hexapod affect the correction, which is defined by the minimum motions of each axis, which
+is ~0.3um. However, we have seen instabilities of ~1um of motion. Using the sensitivity matrix, 1um of motion converts
+to WFEs of ~4.4nm of defocus, and 0.16nm of coma in each direction.
+
+Due to the above discussion, we encourage that **no corrections be applied for motions below 2um in Z and 50 um in X
+and Y.**
+These values correspond to ~10nm of WFE. This is done in order to reduce unnecessary wear on the hexapod and cause
+unnecessary heating.
+
+Another aspect of the CWFS solution which is not yet well understood is strong variations in astigmatism measurements.
+The only mechanism to have any control on astigmatism is to adjust the pressure under the mirror, however testing has
+shown that small corrections are not easily achieved. In the figure below there is an example of a large coefficient
+term for astigmatism. These are believed to be fitting issues as they can vary strongly from image to image. At the
+time of writing this issue is still be tracked down.
+
+
+.. figure:: /_static/cwfs_fit_example.jpg
+    :name: fig-fit-example
+
+    This shows what is believed to be a false measurement in astigmatism, which will often vary significantly
+    from image to image. In this case, only a focus offset should be applied as the coma values are acceptable.
+
+LatissCWFSAlign script
+======================
+
+To collimate and focus the Auxiliary telescope, the scriptQueue script
+`LatissCWFSAlign <https://github.com/lsst-ts/ts_externalscripts/blob/develop/python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py>_`
+is used. The script does not require the scriptQueue and is most easily run from a notebook. The script proceeds
+from a "best focus" position, which can be a rough guess is required, then pistons the hexapod by a set amount,
+generally +/-0.8mm, which corresponds to +/-32.8 mm at the focal plane. Testing has shown this to be an adequate amount,
+balancing the amount of data required to calculate
+the low order Zernikes with keeping the image cutout size small to reduce the overhead of calculating the result.
+
+The CWFS calculation is currently performed using the package developed by Bo Xin, and is currently located
+`here <https://github.com/bxin/cwfs>`_. Note that this package (and our script) outputs Annular Zernike Noll
```

Comment:
I think you need to add double "_" when you do an inline link like this. A single "_" creates a reference that can be called later. 

https://developer.lsst.io/restructuredtext/style.html#external-links

Reply:
all links modified to have a double-underscore at the end (not double quotes), per your link.